### PR TITLE
Feature/craft 3 port

### PR DESCRIPTION
**Pull request by peterbedor**

### CODE REVIEW

File: src/assetbundles/presto/PrestoAsset.php [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<?php
```
Comment (peterbedor) [inline src/assetbundles/presto/PrestoAsset.php]: @thechrisroberts I just fixed the formatting of this file.

File: .editorconfig [hunk 2]
```diff
@@ -5,7 +5,7 @@ charset = utf-8
 end_of_line = lf
 insert_final_newline = false
 indent_size = 4
-indent_style = tab
+indent_style = space
```
Comment (thechrisroberts) [inline .editorconfig:8]: Still so sad.

File: src/config.php [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+ * and make your changes there to override default settings.
+ *
+ * Once copied to 'craft/config', this file will be multi-environment aware as
+ * well, so you can have different settings groups for each environment, just as
+ * you do for 'general.php'
+ */
+
+return [
+
+    // This controls blah blah blah
```
Comment (thechrisroberts) [inline src/config.php]: I'm assuming something is missing here

File: src/services/PrestoService.php [hunk 4]
```diff
@@ -0,0 +1,281 @@
...
+     * @param array $config
+     * @throws \yii\base\ErrorException
+     */
+    public function writeCache($host, $path, $html, $config = [])
+    {
+        if (! Craft::$app->user->isGuest) {
+            return;
+        }
+
+        // TODO: Check if this check is necessary
```
Comment (thechrisroberts) [inline src/services/PrestoService.php]: Did this check check get checked? Or is this to check in the future on checking the check check?

**Pull request closed without merging**